test/basic/br_if.wast

```diff
@@ -0,0 +1,33 @@
+;; tests for eqz and br_if unifications
+
+;; in this case eqz and br_if should not be unified
+(module
+  (func (export "test1") (param $input i32) (result i32)
+    (local $a i32)
+    i32.const 21
+    i32.const 0
+    i32.eqz
```

Comment:
The result of eqz should be returned. Also, `i32.const, br_if` should be tested as well, since i32.const is also eliminated